ansible-ipi-install/inventory/hosts.sample

```diff
@@ -42,6 +42,10 @@ version=""
 # Empty value results in playbook failing with error message.
 build=""
 
+# Provisioning IP Network and dhcp range (default value)
+# prov_network=172.22.0.0/21
```

Comment:
This should be added to the documentation for the playbook too, Can you attach to this PR the relevant modifications?

https://openshift-kni.github.io/baremetal-deploy/latest/Ansible%20Playbook%20Install.html#ansible-playbook-modifying-the-inventoryhosts-file


Thanks

Reply:
Yes, updated.